src/include/planner/plannodes/aggregate_plan_node.h

```diff
@@ -119,7 +119,7 @@ class AggregatePlanNode : public AbstractPlanNode {
   /**
    * @return pointer to predicate for having clause
    */
-  const std::shared_ptr<parser::AbstractExpression> &GetHavingClausePredicate() const {
+  const std::unique_ptr<parser::AbstractExpression> &GetHavingClausePredicate() const {
```

Comment:
This should be returning `std::ManagedPointer<parser::AbstractExpression>`

Reply:
Search and replace strikes again. Thanks!